RecoHI/HiCentralityAlgos/python/HiCentrality_cfi.py

```diff
@@ -38,7 +38,8 @@
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
 for e in [pp_on_XeXe_2017, pp_on_AA_2018]:
     e.toModify(hiCentrality,
-               producePixelTracks = False,
+               producePixelTracks = True,
+               srcPixelTracks = cms.InputTag("hiConformalPixelTracks"),
                srcTracks = cms.InputTag("generalTracks"),
                srcVertex = cms.InputTag("offlinePrimaryVertices")
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.